Himalf/GroupPortfolio
# Add skills section in Home page

### What does this PR do?
This PR adds a new "Skills" section to the homepage, displaying the front-end, back-end,  essential skills and soft skills. The section includes icons for each skill and organizes them into four categories.

### How to test
1. Pull the branch `feature/skills-section` from the remote repository.
2. Navigate to the homepage and verify that the "Skills" section appears correctly.
3. Ensure that the skills are displayed under the correct categories (Front-End, Back-End, Essential).
4. Check that the icons are visible next to the skill names.

### Additional notes
- The section is designed to be responsive using Tailwind CSS.
- Ensure that all links and navigation in the header are working correctly after merging.
